### PR TITLE
revert to lodash 3.8.0 version

### PR DESCRIPTION
**Pull request by david-meza**
Temporary fix to lodash 4.0 release.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "angular": "1.2 - 1.4",
```
Comment (nmccready) [inline bower.json]: just make it 3.X

Comment (nmccready) [inline bower.json]: I'll merge on that

**Comment by david-meza** [comment]
Done!

**Comment by nmccready** [comment]
TY LGTM

**Pull request merged**